src/libData/AccountData/AccountStoreSC.cpp

```diff
@@ -1610,6 +1610,16 @@ void AccountStoreSC<MAP>::CleanNewLibrariesCache() {
   m_newLibrariesCreated.clear();
 }
 
+template <class MAP>
```

Comment:
Doesn't it mean that because we have two instances of this, one with `std::map`, one with `std::unordered_map`, we'll get two instances of this method, eventually called twice, into two different members variables (so far everything is good),  and ... attempting to bind to the same UNIX domain socket (which will likely fail for the 2nd instance).